extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonUtils.java

```diff
@@ -108,6 +109,12 @@ public static void setHttpClientOptions(OidcCommonConfig oidcConfig, TlsConfig t
         if (proxyOpt.isPresent()) {
             options.setProxyOptions(proxyOpt.get());
         }
+
+        OptionalInt webClientMaxPoolSize = oidcConfig.webClientMaxPoolSize;
+        if (webClientMaxPoolSize.isPresent()) {
```

Comment:
@glefloch It is here where the problem is - those tests are using the dynamic `OidcTenantConfig` - so it is not injected, and is `null` - so please initialize this field to an empty `OptionalInt`